scenarioo-client/test/e2e/pages/stepPage.ts

```diff
@@ -74,6 +74,11 @@ class StepPage {
         await expect(element(by.id('fallbackMessage')).isDisplayed()).toBeFalsy();
     }
 
+    async assertErrorResponseIsShown() {
+        await expect(element(by.id('stepNotFoundErrorResponseMethod')).getText()).toEqual('GET');
```

Comment:
I would put the assertion parameters (e.g. GET and 404 as parameters). I know we only use it once but I prefer to have it visible in the webtest itself rather than the page object. So you get some idea what is asserted.

Reply:
Good idea. I updated this.